### DISCUSSION TITLE
Got SB500 back to Sling dashboard after hard reset

### DISCUSSION THREAD

**Discussion by midcorestudios** [GerryDazoo/Slinger#628] [Q&A]
Ok, so after a period of trial and errors I have managed to get the SB500 back to the sling dashboard where I can see live tv, settings etc.  However there seems to be some config files missing since I only have the option to change network and inputs, I no longer get the whole setup walk through of zip code device etc.   Now I need to figure out how to add those options, I had some luck previously with Slinplayer 5 but since I can no longer get it to load I'm looking for Ideas.   Would anyone with a working 500 be able to zip up their Appdata/Slingplayer folder for me?  I bought my boxes off ebay so never had a chance to log into the player to get the admin passwords etc.

**Comment by cooley420** [comment]
Can you tell me how you got back to the dashboard after a reset?   I’m stuck at the device section of setup and without the sling servers seems to be as far as I can go.   I can still stream with slinger but would like to get back to the dashboard.
